ONSdigital/rm-reporting
# Upgrade sqlalchemy to 2.x

# What and why?

sqlalchemy 2.0 finally released so we upgraded to it to keep current.
As a bonus, we also removed the echo from the engine so the amount of noisy logging should drastically decrease.

# How to test?

- Deploy to environment and run acceptance tests to seed data
- Download response chasing report
- Look at the responses dashboard.  If the data is correct (usually 8 ru refs and 1 account) then it's working.

After this, check the logs and make sure there are no errors.

# Trello
